### PR TITLE
Update README.zh-CN.md about fixing format

### PR DESCRIPTION
**Pull request by fwqaaq**
## Types of changes

- [ ] Docs change / Dependency upgrade
- [x] Bug fix
- [ ] New feature / Improvement
- [ ] Refactoring
- [ ] Breaking change

## Checklist

- [ ] I have added changeset via `pnpm run changeset`.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

File: README.zh-CN.md [hunk 1]
```diff
@@ -36,13 +36,13 @@
 
 ## 参与贡献
 
-> **Rspack 项目目前仍在积极开发中，仍然缺失了很多的 webpack 的插件 Hook 以及一些 API 目前可能不够稳定,我们正在努力补齐这些 API，并不断地提升 Rspack 的稳定性，如果您有任何建议或意见，欢迎随时提交 PR 或 Issue，我们将不胜感激。**
+> **Rspack 项目目前仍在积极开发中，仍然缺失了很多的 webpack 的插件 Hook 以及一些 API 目前可能不够稳定，我们正在努力补齐这些 API，并不断地提升 Rspack 的稳定性，如果您有任何建议或意见，欢迎随时提交 PR 或 Issue，我们将不胜感激。**
 
 请阅读 [贡献指南](./CONTRIBUTING.md) 来共同参与 Rspack 的建设。
```
Comment (fwqaaq) [inline README.zh-CN.md:41]:
```suggestion
请阅读[贡献指南](./CONTRIBUTING.md)来共同参与 Rspack 的建设。
```

**Review by hardfist — APPROVED** [review]

**Comment by hardfist** [comment]
please sign CLA

**Comment by fwqaaq** [comment]
> please sign CLA

Done! Thank you!

File: README.zh-CN.md [hunk 2]
```diff
@@ -57,15 +57,15 @@
...
 感谢:
 
-- [webpack 团队和社区](https://webpack.js.org/) 创建了一个优秀的打包工具和丰富的生态。
+- [webpack 团队和社区](https://webpack.js.org/)创建了一个优秀的打包工具和丰富的生态。
 - [@sokra](https://github.com/sokra) 在 [webpack](https://github.com/webpack/webpack) 项目上的出色工作。
 - [@ScriptedAlchemy](https://github.com/ScriptedAlchemy) 创造了模块联邦，并帮助 Rspack 与社区建立联系。
 - [SWC](https://swc.rs/) 项目（由 [@kdy1](https://github.com/kdy1) 创建），为 Rspack 的代码解析、转换和压缩提供了支持。
 - [esbuild](https://github.com/evanw/esbuild) 项目（由 [@evanw](https://github.com/evanw) 创建），它启发了 Rspack 的并发架构。
 - [NAPI-RS](https://github.com/napi-rs/napi-rs) 项目（由 [@Brooooooklyn](https://github.com/Brooooooklyn) 创建），为 Rspack 的 node-binding 实现提供了支持。
 - [Parcel](https://github.com/parcel-bundler/parcel) 项目（由 [@devongovett](https://github.com/devongovett)创建）， 它是 Rust Bundler 的先行探索者并启发了 Rspack 的增量构建架构。
```
Comment (fwqaaq) [inline README.zh-CN.md:66]:
```suggestion
- [Parcel](https://github.com/parcel-bundler/parcel) 项目（由 [@devongovett](https://github.com/devongovett)创建），它是 Rust Bundler 的先行探索者并启发了 Rspack 的增量构建架构。
```

**Review by bvanjoi — APPROVED** [review]

**Pull request merged**